test/stream/api/base/api-startup/expected-server-stdout

```diff
@@ -23,13 +23,13 @@
 {"level":"info","message":"exiting (terminated)"}
 {"level":"info","message":"shutting down HTTP server at \"%s\"","component":"api"}
 {"level":"info","message":"HTTP server shutdown finished","component":"api"}
+{"level":"info","message":"stopping volumes registration","component":"volumes.registry"}
+{"level":"info","message":"closing etcd session: context canceled","component":"volumes.registry.etcd.session"}
+{"level":"info","message":"closed etcd session","component":"volumes.registry.etcd.session"}
+{"level":"info","message":"stopped volumes registration","component":"volumes.registry"}
 {"level":"info","message":"closing volumes","component":"storage.node.writer"}
 {"level":"info","message":"closing volume","volume.path":"%s/hdd/my-volume","volume.id":"my-volume","component":"storage.node.writer.volume"}
 {"level":"info","message":"closed volume","volume.path":"%s/hdd/my-volume","volume.id":"my-volume","component":"storage.node.writer.volume"}
-{"level":"info","message":"received shutdown request","component":"volumes.registry"}
-{"level":"info","message":"closing etcd session: context canceled","component":"volumes.registry.etcd.session"}
-{"level":"info","message":"closed etcd session","component":"volumes.registry.etcd.session"}
-{"level":"info","message":"shutdown done","component":"volumes.registry"}
 {"level":"info","message":"closing volumes stream","component":"volume.repository"}
```

Comment:
Fixed order issue, at first stop registration (let know other nodes, the volumes are no more available), then close volumes.